tests/src/test/scala/mhtml/tests.scala

```diff
@@ -352,15 +352,6 @@ class Tests extends FunSuite {
     assert(w == List("Unknown element yolo. Did you mean col instead?"))
   }
 
-  test("Atom warn") {
```

Comment:
Do you think you could make a few negative test cases with [assertTypeError](http://doc.scalatest.org/3.0.0/index.html#org.scalatest.Assertions@assertTypeError(code:String)(implicitpos:org.scalactic.source.Position):org.scalatest.Assertion)?